docker-compose.yml

```diff
@@ -120,8 +120,8 @@ services:
       - ./backoffice/.envs/local/.django
       - ./backoffice/.envs/local/.postgres
     ports:
-      - '8001:8001'
```

Comment:
if you want to change the ports. they will also have to change in more places:

backoffice/.envs/local/.django
`FRONTEND_LOGIN_SUCCESS=http://localhost:8080/backoffice` to use new port of ui

`workflows/scripts/connections/connections.json` to use new port of backoffice
`backoffice_conn` to use the new port